src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -276,6 +277,18 @@ function MoneyRequestView({
         [transactionAmount, isSettled, isCancelled, isPolicyExpenseChat, isEmptyMerchant, transactionDate, hasErrors, hasViolations, translate, getViolationsForField],
     );
 
+    const deleteTransaction = useCallback(() => {
+        if (!parentReportAction) {
+            return;
+        }
+        const iouTransactionID = parentReportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? parentReportAction.originalMessage?.IOUTransactionID ?? '-1' : '-1';
+        if (ReportActionsUtils.isTrackExpenseAction(parentReportAction)) {
+            IOU.deleteTrackExpense(parentReport?.reportID ?? '-1', iouTransactionID, parentReportAction, true);
+            return;
+        }
+        IOU.deleteMoneyRequest(iouTransactionID, parentReportAction, true);
+    }, [parentReport?.reportID, parentReportAction]);
```

Comment:
I think there is no point in using `useCallback` here since you are not passing down this function to another component and you are not using it in another function wrapped in `useCallback`. 

 Can you instead define this function outside of the component and receive as parameters `parentReport` and `parentReportAction` 🙏 ?

Then later you call it with `deleteTransaction(parentReport, parentReportAction)`

Reply:
@aldo-expensify please check again